imagemodal/mixins/fragment.py

```diff
@@ -0,0 +1,49 @@
+"""
+Mixin fragment/html behavior into XBlocks
```

Comment:
Ultimately, this should be moved to a 3rd party repo, like `xblock-utils-stanford`
so that this behavior can be shared between all of our xblocks..
Until then, it's in it's own mixin file here.

One notable difference between this and `free-text-response` (from which this is cribbed)
is that the `build_fragment` method now handles all template/fragment logic.

In `free-text`, you had to pre-render the template and then pass the output along to `build_fragment`.
Now it's all-in-one :)
You just tell it which template, which context, and which static files,
and it does the rest :D

Reply:
Maybe?
Part of it is this isn't the extent of it;
it's just the only other mixin for this xblock.
We have others, like `ScorableXBlock` or something in `free-text`.

I'm not actually arguing for any particular implementation yet,
just flagging this as code that should definitely be shared w/ all of our xblocks
instead of copy/pasting it everywhere like we're doing now :)